packages/manager/apps/dedicated/client/app/ip/components/list/firewall/ip-ip-firewall.controller.js

```diff
@@ -217,6 +220,16 @@ export default /* @ngInject */ function IpFirewallCtrl(
         },
       )
       .finally(() => {
+        // Sort list to display deny action first
+        self.rules.list.results = self.rules.list.results.sort((a, b) => {
```

Comment:
could fail if ` IpFirewall.getFirewallRules` fails. Check if object is not null before sorting